src/dune

```diff
@@ -1,5 +1,6 @@
 (executable
  (name vscode_ocaml_platform)
+ (modules :standard \ vscode_ocaml_platform_web)
```

Comment:
I'm expecting that when we want to add features to the web extension, this won't reuse anything in the existing code base. I'd maybe suggest we put this in a separate directory (e.g. `src-browser`) to avoid any confusion.

Reply:
I agree